### DISCUSSION TITLE
Add toggle for Snap Color in settings

### DISCUSSION THREAD

**Discussion by 177013IsNice** [Quaver/Quaver#4781] [Feature Request]
### Is your feature request related to a problem? Please describe.

I feel like given the lack of skinning knowledge of the community, it would be nice to include a button in settings to toggle snap colors but only if the skin contains snap colored variants of hitobjects. I personally feel this would fit nicely under the Configuration section in the Skins setting. It just feels like a nice simple quality of life feature that makes Quaver feel even nicer.

### Describe the solution you'd like

To make sure all skins are compatible, the switch would only be visible if the skin either contains a spritesheet for snap colors, or at least has snap colored variants of the hitobjects.

### Describe alternatives you've considered

The current method of telling people to go to the .ini and change it works but only to a small extent, since to people who don't skin, the ini looks like a wall of text.

**Comment by EpiCreeper** [comment]
i know im late but this please
